webservices/legal_docs/current_murs.py

```diff
@@ -8,6 +8,8 @@
 from webservices.rest import db
 from webservices.utils import create_eregs_link, get_elasticsearch_connection
 from webservices.tasks.utils import get_bucket
+from webservices.legal_docs.utils import get_subjects, get_election_cycles, get_participants, get_sorted_respondents, get_disposition, get_documents, get_open_and_close_dates
+from webservices.legal_docs.utils import STATUTE_REGEX, REGULATION_REGEX
```

Comment:
These imports are not used. There are others as well that should be removed - `re, create_eregs_link, get_sorted_repsondents`, etc